transforms/remove-binding/index.js

```diff
@@ -0,0 +1,51 @@
+const { getParser } = require('codemod-cli').jscodeshift;
+
+module.exports = function transformer(file, api) {
+  const j = getParser(api);
+
+  const root = j(file.source);
+
+  const node = root.find(j.ObjectProperty, {
+    key: {
+      type: 'Identifier',
+    },
+  });
+
+  let filteredNodes = node.filter((path) => {
+    return path.value.key.name && path.value.key.name.endsWith('Binding');
+  });
+
```

Comment:
💅🏽 I would early return here to safe some execution time:

```suggestion
if(!filteredNodes.length) {
   return node.toSource();
}
```